test/cypress/utils/enterSecondPassphrase.js

```diff
@@ -1,12 +1,22 @@
-const ss = {
+import ss from '../../constants/selectors';
+
+const ss2 = {
```

Comment:
don't like the name, but also, not sure what we could but instead of this 😕

Reply:
correct, I just fixed